### PR TITLE
A refresher for the 'phantom data' article.

### PR DESCRIPTION
**Pull request by allipatev**

### CODE REVIEW

File: Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md [hunk 1]
```diff
@@ -47,18 +43,18 @@ FROM
 ORDER BY
     MEASURE_TIME DESC;
 ```
-Huge phantom percentage (>5%) for a longer periods of time is not normal.
+Huge phantom percentage (>5%) for longer periods of time is not normal.
```
Comment (gabimoruz) [inline Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md:46]: I would say here that ups and downs are normal, a consistent upwards trend is not.

File: Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md [hunk 2]
```diff
@@ -32,7 +28,7 @@ SELECT
     MEM_SIZE,
     MULTICOPY_SIZE,
     COMMIT_SIZE-MEM_SIZE-MULTICOPY_SIZE                    PHANTOM_SIZE,
-    CAST((local.PHANTOM_SIZE)*100/COMMIT_SIZE AS DEC(7,3)) "PHANTOM_%"
+    CAST((local.PHANTOM_SIZE)*100/(MEM_SIZE+MULTICOPY_SIZE) AS DEC(7,3)) "PHANTOM_%"
```
Comment (gabimoruz) [inline Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md:31]: The query is the one from https://exasol.atlassian.net/browse/EXASOL-2335 , is this correct?

**Review by gabimoruz — CHANGES_REQUESTED** [review]
The comments about the fact that the upward trend is what really counts should be included I think. Other than that the other comments are not that important.

File: Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md (hunk 1 above)
Comment (allipatev) [reply Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md:46]: How about

> Huge phantom percentage (>5%) for longer periods of time and especially consistent upwards trend are not normal.

?

Just "consistent upwards trend" doesn't look measurable enough and I, as a Support agent, will need to be able to explain it in a crystally clear way.

File: Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md (hunk 2 above)
Comment (allipatev) [reply Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md:31]: As far as I can see the query to be changed (fixed) is the same as in that roadmap ticket.

**Comment by allipatev** [comment]
@gabimoruz could you take a look at my response before I commit the changes.

**Review by gabimoruz — APPROVED** [review]

**Comment by allipatev** [comment]
The requested changes were applied.

**Pull request merged**